strapi/strapi
# 🐞 fix: admin-user-filters-and-sort

### What does it do?

The issue with filtering and sorting based on email and Active User in the settings page on the Admin panel has been resolved.

### Why is it needed?

Previously, the Users page in the admin panel had email and active user filters and sorting, but there was an issue with these functionalities. When either of the filters / sorting was applied, it would crash and return an error.

### How to test it?

To check whether the filtering functionality is working properly with the email and active user filters, follow these steps in the Admin Panel: 

1. Go to Settings.
2. Navigate to the Administration Panel.
3. Click on Users.

Once you have reached the Users page, test the email and active user filters to ensure they are functioning correctly.

### Related issue(s)/PR(s)

This update fixes the issue reported in [16554](https://github.com/strapi/strapi/issues/16554).
